### DISCUSSION TITLE
Apakah AdblockID dapat digunakan pada ekstensi Adblock Plus?

### DISCUSSION THREAD

**Discussion by arfshl** [realodix/AdBlockID#1110] [Q&A] [answered]
Apakah AdblockID dapat digunakan pada ekstensi Adblock Plus?

**Comment by realodix** [comment, marked as answer]
Bisa, tetapi diikuti dengan kemampuan AdBlockID yang sangat menurun drastis hingga di bawah 50%.

AdBlockID sangat bergantung pada teknologi penghilang iklan yang lebih maju, yang sayangnya saat ini diketahui hanya ada di uBlock Origin dan AdGuard.

**Reply by Jokopentil** [reply]
Ublock origin terdapat banyak masalah untuk blokir iklan gif atau iklan gambar anaimasi
Apakah bisa riques buat filter yang cocok khusus buat ublock origin🙏🙏🙏 tolong di jawab

**Reply by Jokopentil** [reply]
Untuk adblock plus sangat bagus blokir iklan gambar animasi
Tapi ada kelemahan terkadang tak bisa blokir anti adblock walaupun pakai tampermonket

**Reply by realodix** [reply]
> Apakah bisa riques buat filter yang cocok khusus buat ublock origin🙏🙏🙏 tolong di jawab

Pertanyaan bagus, mohon dibuat diskusi terpisah agar orang lain dapat dengan mudah mecarinya 🙏

**Reply by 3xploiton3** [reply]
> Untuk adblock plus sangat bagus blokir iklan gambar animasi Tapi ada kelemahan terkadang tak bisa blokir anti adblock walaupun pakai tampermonket

kenapa harus pakai adblock plus? apakah ada ketergantungan dari environment device?
gunakan saja saran dari master @realodix ,
saya gunakan keduanya dan keduanya bagus tp sekarang lebih sering menggunakan ublock-origin

**Reply by Jokopentil** [reply]
Iklan animasi biasanya yang paling banyak iklan judi dan game judi, iklan obat kuat dan dating porn
Biasanya pakai abpindo pakai adblock plus,  adguard bisa di block
Tapi pakai ublock origin tidak terblokir
Walau pakai adblockid sekalipun
Klo bisa riquest buat khusus ublock origin

**Reply by realodix** [reply]
Ini sebenarnya out of topic, karena disini judulnya tentang "Menggunakan AdBlockID di AdBlock Plus", sementara yg Anda tanyakan tentang filter khusus di uBO. Saya sangat menyarankan untuk membuat diskusi baru, karena Saya khawatir jika seseorang menanyakan hal yang sama dengan Anda, mereka akan sulit menemukan jawabannya.
